parsedatetime/pdt_locales/__init__.py

```diff
@@ -6,22 +6,25 @@
 All of the included locale classes shipped with pdt.
 """
 
-try:
-    import PyICU as pyicu
-except:
-    pyicu = None
+from __future__ import absolute_import
+from .icu import get_icu
 
+locales = ['de_DE', 'en_AU', 'en_US', 'es', 'nl_NL', 'pt_BR', 'ru_RU']
```

Comment:
I struggled between the auto-discover trick and this for a while. But finally I feel like `import this` said that "explicit is better than implicit.". So let's just keep this **simple and stupid** even it's a bit redundant.